src/_gettsim/functions/policy_function.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import functools
+import inspect
+from collections.abc import Callable
+from datetime import date
+from typing import Any, TypeVar
+
+import numpy
+
+T = TypeVar("T")
+
+
+class PolicyFunction(Callable):
```

Comment:
Even though the coverage report is probably nonsense, wouldn't a set of tests for setting up the instances of the class be useful?